airbyte-workers/src/test/java/io/airbyte/workers/DefaultReplicationWorkerTest.java

```diff
@@ -135,23 +140,34 @@ void test() throws Exception {
     verify(destination).accept(RECORD_MESSAGE2);
     verify(source).close();
     verify(destination).close();
+    verify(recordSchemaValidator).validateSchema(RECORD_MESSAGE1, syncInput);
+    verify(recordSchemaValidator).validateSchema(RECORD_MESSAGE2, syncInput);
   }
 
   @Test
-  void testSourceNonZeroExitValue() throws Exception {
```

Comment:
Is there a reason you replaced this test instead of just adding a new one and leaving this one as-is?

Reply:
oops